### PR TITLE
Add capital-framework to repo's root, rewrite readme and contributing docs

### PR DESCRIPTION
**Pull request by contolini**
## Changes

- [`capital-framework`](https://github.com/cfpb/capital-framework) is now in this repo's root instead of `src/`
- Purged capital framework language from both [`README.md`](https://github.com/cfpb/design-system/pull/272/files#diff-04c6e90faac2675aa89e2176d2eec7d8) and [`CONTRIBUTING.md`](https://github.com/cfpb/design-system/pull/272/files#diff-04c6e90faac2675aa89e2176d2eec7d8) and modified all the steps to include the new `@cfpb` npm scope.
- Updated the docs' website [build script](https://github.com/cfpb/design-system/pull/272/files#diff-d34809464730fe7381f1acde5e58c8e1R33) to load the capital framework stylesheets that are now in the project's root.

## Testing

Everything should work the same way it always has.

1. Pull down this branch.
1. `cd docs/`
1. `yarn`
1. `yarn start`
1. View the design system website at http://localhost:4000/design-system/

## Notes

- I have not yet changed all the CF components to use `@cfpb/` instead of `cf-`. We are still finalizing that decision with the FEWDs. Plus, it makes sense to do that at the very end of this project to avoid having to redo it if the capital-framework repo changes in the meantime.

### CODE REVIEW

**Comment by cfarm** [comment]
>     I have not yet changed all the CF components to use @cfpb/ instead of cf-. We are still finalizing that decision with the FEWDs. Plus, it makes sense to do that at the very end of this project to avoid having to redo it if the capital-framework repo changes in the meantime.

Is this why all these packages are 'not found' in the readme?

Screen Shot 2019-08-23 at 4 01 45 PM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
+All contributions to this project will be released under the CC0 public domain
+dedication. By submitting a pull request or filing a bug, issue, or
+feature request, you are agreeing to comply with this waiver of copyright interest.
+Details can be found in our [TERMS](https://github.com/cfpb/design-system/blob/master/TERMS.md) and [LICENSE](https://github.com/cfpb/design-system/blob/master/LICENSE).
```
Comment (cfarm) [inline CONTRIBUTING.md]: TERMS link is broken, should link to a version of this file https://github.com/cfpb/open-source-project-template/blob/master/TERMS.md
LICENSE is wrong, says MIT, should be our own custom license from https://github.com/cfpb/open-source-project-template/blob/master/LICENSE

File: README.md [hunk 2]
```diff
@@ -1,38 +1,89 @@
...
-- Custom Netlify CMS widgets are in `docs/admin/src/`.
-- Jekyll templates are in `docs/_layouts/`.
+Please view our [contribution guidelines.](CONTRIBUTING.md)
 
-### Getting started with Capital Framework
 
-We're in the process of migrating Capital Framework to this repository.
-For now, it still lives at [cfpb/capital-framework](https://github.com/cfpb/capital-framework). Stay tuned.
+## Open source licensing info
+1. [TERMS](TERMS.md)
```
Comment (cfarm) [inline README.md]: same as above comment

File: CONTRIBUTING.md (hunk 1 above)
Comment (contolini) [reply CONTRIBUTING.md]: These links should resolve once these changes reach master.

**Pull request merged**